examples/01_Simulation.ipynb

```diff
@@ -342,6 +342,551 @@
+    "\n",
+    "In this example we will repeat the Piecewise load example, this time using the GaussianNoiseLoadWrapper to represent our uncertainty in our future load estimate. \n",
+    "\n",
+    "First we will import the necessary classes and construct the Piecewise load estimation just as in the previous example."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from progpy.loading import Piecewise, GaussianNoiseLoadWrapper\n",
+    "loading = Piecewise(\n",
+    "        InputContainer=m.InputContainer,\n",
+    "        times=[600, 900, 1800, 3000],\n",
+    "        values={'i': [2, 1, 4, 2, 3]})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next we will wrap this loading object in our Gaussian noise load wrapper"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "loading_with_noise = GaussianNoiseLoadWrapper(loading, 0.2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this case we're adding guassian noise with a standard deviation of 0.2 to the result of the previous load estimator.\n",
+    "\n",
+    "Now let's simulate and look at the input profile."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "results = m.simulate_to_threshold(loading_with_noise, save_freq=100)\n",
+    "fig = results.inputs.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note the loading profile follows the piecewise shape, but with noise. If you run it again, you would get a slightly different result."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "results = m.simulate_to_threshold(loading_with_noise, save_freq=100)\n",
+    "fig = results.inputs.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here are the corresponding outputs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = results.outputs.plot(compact=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that the noise in input can be seen in the resulting output plots.\n",
+    "\n",
+    "The seed can be set in creation of the GaussianNoiseLoadWrapper to ensure repeatable results, for example."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "loading_with_noise = GaussianNoiseLoadWrapper(loading, 0.2, seed=2000)\n",
```

Comment:
This block also gives me an error, "__init__() got an unexpected keyword argument 'seed'"